frontend/src/app/annual-project-report/[year]/mlfs/workspace/page.tsx

```diff
@@ -0,0 +1,477 @@
+'use client'
```

Comment:
No need for directives, we use React 18 without RSC (https://18.react.dev/reference/rsc/directives). Even if we upgrade to 19 we most likely won't use RSC.